alvr/filesystem/src/lib.rs

```diff
@@ -106,8 +106,6 @@ pub struct Layout {
 
 impl Layout {
     pub fn new(root: &Path) -> Option<Self> {
-        let root = root.canonicalize().ok()?;
-
```

Comment:
Given that now Layout::new() is now infallible, the Option<> should be removed from the return type. Check the changes of the faulty commit to see all the places that should be reverted